src/lib/WellKnown.ts

```diff
@@ -12,3 +12,28 @@ export const WellKnownTypesMap: { [key: string]: string } = {
     "google/protobuf/type.proto": "google-protobuf/google/protobuf/type_pb",
     "google/protobuf/wrappers.proto": "google-protobuf/google/protobuf/wrappers_pb",
 };
+
+export type WellKnownExtension = { declarations?: string[], extensions: { [key: string]: string[] } };
+
+export const WellKnownExtensionsMap: { [key: string]: WellKnownExtension } = {
+    "google/protobuf/timestamp.proto": {
+        "extensions": {
+            "Timestamp": [
+                "toDate(): Date;",
+                "fromDate(date: Date): null;",
+                "static fromDate(date: Date): Timestamp;",
```

Comment:
I think we need move those functions into templates

Reply:
Could you advise on the best structure for that? Are you imagining partials for each of the messages within each of the well known types? I could imagine for example:

```
template
  partial
    wellKnown
      timestamp
        declarations.hbs
        extensions
          [messageName].hbs
```

I was primarily concerned about how these would be passed conditionally into the messages in a way that is not too obscenely complicated.